src/components/FlexLayout/FlexLayout.options.tsx

```diff
@@ -1,5 +1,6 @@
-export const AlignItems = [ 
```

Comment:
Is this component missing a index.tsx file?

Reply:
🙈